assets/css/admin.scss

```diff
@@ -17,6 +17,96 @@ a.wpjm-activate-licence-link:active {
 	color: orangered;
 }
 
+/** Promote Job Modal **/
+promote-job-modal {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+	padding: 30px 80px 50px 80px;
+	img {
```

Comment:
Another thing I was thinking here is that it would be interesting to give more CSS classes to the elements. Usually, it's a good practice, but mainly here because we're doing it a little more dynamic getting the content from wpjobmanager.com. So if we decide to add any other image to any part, for example, it would have a `width: 100%`.

The same would apply to the button, list, and so on.

Reply:
Done in f4cad966c8698eddfaa04a6b5425e2a08e32d047